src/components/input/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { InputStyle } from "./style";
+
+export const Input = ({ placeholder, ...props }) => {
+
+    return (
+        <InputStyle 
+            {...props}
+            type='text'
+            placeholder={placeholder}
```

Comment:
I don't know why you have passed the placeholder property separately, but you can pass every properties by using just the `props`. so if you need to pass the placeholder attribute you pass at the usage moment and not here in the declaration of the component